src/main/kotlin/com/sourcegraph/cody/autocomplete/CodyAutocompleteManager.kt

```diff
@@ -155,6 +157,11 @@ class CodyAutocompleteManager {
     if (isTriggeredImplicitly && !autoCompleteDocumentContext.isCompletionTriggerValid()) {
       return
     }
+    if (isTriggeredExplicitly &&
+        CodyAuthenticationManager.instance.getActiveAccount(project) == null) {
+      HintManager.getInstance().showErrorHint(editor, "Cody: No account signed-in")
```

Comment:
It looks like this:

![image](https://github.com/sourcegraph/jetbrains/assets/19799111/f5e56511-96a1-4b18-9b52-2209bec07474)